src/VisualStudio/Xaml/Impl/Implementation/LanguageServer/Extensions/SymbolExtensions.cs

```diff
@@ -53,16 +54,17 @@ public static async Task<IEnumerable<TaggedText>> GetDescriptionAsync(this ISymb
                 return Enumerable.Empty<TaggedText>();
             }
 
-            var textContentBuilder = new List<TaggedText>();
-            textContentBuilder.AddRange(symbol.ToDisplayParts(s_descriptionStyle).ToTaggedText());
-
-            var documentation = symbol.GetDocumentationParts(semanticModel, offset, formatter, cancellationToken);
-            if (documentation.Any())
+            var quickInfo = await QuickInfoUtilities.CreateQuickInfoItemAsync(codeProject.Solution.Workspace, semanticModel, TextSpan.FromBounds(0, 0), ImmutableArray.Create(symbol), cancellationToken).ConfigureAwait(false);
```

Comment:
```suggestion
            var quickInfo = await QuickInfoUtilities.CreateQuickInfoItemAsync(codeProject.Solution.Workspace, semanticModel, span: default, ImmutableArray.Create(symbol), cancellationToken).ConfigureAwait(false);
```

(up to you on what you prefer here).